test/tortoise/connection/backoff_test.exs

```diff
@@ -11,8 +11,8 @@ defmodule Tortoise311.Connection.BackoffTest do
     assert {^min, backoff} = Backoff.next(backoff)
     {_, backoff} = Backoff.next(backoff)
     assert {^max, backoff} = Backoff.next(backoff)
-    # should roll back to min interval now
-    assert {^min, _} = Backoff.next(backoff)
+    # should NOT roll back to min interval now
+    assert {^max, _} = Backoff.next(backoff)
```

Comment:
Wow, this really looks like an intentional decision to roll the backoff back to the minimum. I'm a little curious about why, but I agree with this change to not do that.

Reply:
https://en.wikipedia.org/wiki/Exponential_backoff

"The 'truncated' simply means that after a certain number of increases, the exponentiation stops; i.e. the retransmission timeout reaches a ceiling, and thereafter does not increase any further."

Cycling backoffs is apparently NOT a thing.